templates/JazzyFile.stencil

```diff
@@ -0,0 +1,22 @@
+author: Microsoft
+author_url: https://azure.github.io/azure-sdk/
+github_url: https://github.com/Azure/azure-sdk-for-ios
+module: {{ model.name }}
+module_version: 0.1
```

Comment:
Do we need to pull this value from swagger file 's API version?

Reply:
it won't be the swagger's API versions, but it does need to be dynamic.